libr/core/cmd_type.c

```diff
@@ -383,7 +383,7 @@ static void save_parsed_type_size(RCore *core, const char *parsed) {
 	}
 }
 
-static void save_parsed_type(RCore *core, const char *parsed) {
+R_API void save_parsed_type(RCore *core, const char *parsed) {
```

Comment:
this function name cant be public. rename it to something meaningful like r_core_something, but this looks like a completely random function name to make it public. and also, this function is only used from the same file, so there's no need to make it public